io.github.TheWisker.Cavasik.json

```diff
@@ -0,0 +1,83 @@
+{
+    "app-id" : "io.github.TheWisker.Cavasik.desktop",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "cavasik",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--socket=pulseaudio",
+        "--device=dri"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+    	"shared-modules/linux-audio/fftw3f.json",
```

Comment:
you haven't added the shared-modules.

Reply:
What do you mean exactly?, I only use that one module.

Reply:
From the README on the repo:

https://github.com/flathub/shared-modules/

![image](https://github.com/flathub/flathub/assets/114441/935ef1d8-f071-4bb2-a703-71ebf134e9b8)

Reply:
you didn't do that.

Reply:
You have to add https://github.com/flathub/shared-modules as submodule, or the builder will not find the linked modules

Reply:
Ok, on it.